### DISCUSSION TITLE
A Question for niels747 ⚡

### DISCUSSION THREAD

**Discussion by Meso22** [niels747/2D-Weather-Sandbox#31] [Q&A] [answered]
What are you planning on making the lightning look like?

**Comment by niels747** [comment, marked as answer]
⚡ It will be very difficult to make lightning look realistic, I will start by creating randomly generated yellow lines with branches. Then add some blooming to make it look better. Eventually I will see if I can actually simulate electrical charges...

**Reply by Meso22** [reply]
I can't wait to see it! It would make storms feel more alive and active.
